<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PENG_InsulationDialog</class>
 <widget class="QDialog" name="PENG_InsulationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget4">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabHC">
      <attribute name="title">
       <string>Heat Conservation (Outdoor)</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel1">
         <property name="text">
          <string>Click cell to select, [F2] to edit data (temperature is horizontal [째C], line size is vertical [NomOD inch], insulation in [mm])</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="RB_TableView" name="tvHeatConservationOutdoor"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabHCI">
      <attribute name="title">
       <string>Heat Conservation (Indoor)</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel1_4">
         <property name="text">
          <string>Click cell to select, [F2] to edit data (temperature is horizontal [째C], line size is vertical [NomOD inch], insulation in [mm])</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="RB_TableView" name="tvHeatConservationIndoor"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCC">
      <attribute name="title">
       <string>Cold Conservation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel1_2">
         <property name="text">
          <string>Click cell to select, [F2] to edit data (temperature is horizontal [째C], line size is vertical [NomOD inch], insulation in [mm])</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="RB_TableView" name="tvColdConservation"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPP">
      <attribute name="title">
       <string>Personnel Protection</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel1_3">
         <property name="text">
          <string>Click cell to select, [F2] to edit data (temperature is horizontal [째C], line size is vertical [NomOD inch], insulation in [mm])</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="RB_TableView" name="tvPersonnelProtection"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>439</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RB_TableView</class>
   <extends>QTableView</extends>
   <header>rb_tableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tvHeatConservationOutdoor</tabstop>
  <tabstop>tabWidget4</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbHelp</tabstop>
  <tabstop>tvHeatConservationIndoor</tabstop>
  <tabstop>tvColdConservation</tabstop>
  <tabstop>tvPersonnelProtection</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
